setup.py

```diff
@@ -1,19 +1,25 @@
 from setuptools import setup, find_packages
 
-# Load the contents of the metadata module without using import, since importing requires all dependencies
-# to be available and at this point pip hasn't checked them yet.
+# Load the contents of the metadata module without using import, since
+# importing requires all dependencies to be available and at this point
+# pip hasn't checked them yet.
 metadata = {}
 with open("wmfdata/metadata.py") as file:
     exec(file.read(), metadata)
 
+description = (
+  "Tools for analyzing data on SWAP, a platform for confidential "
+  "Wikimedia data"
+)
+
 setup(
     name="wmfdata",
     version=metadata["version"],
-    description="Tools for analyzing data on SWAP, a platform for confidential Wikimedia data",
+    description=description,
```

Comment:
What is gained by separating this out into a variable?

Reply:
That was a way to reduce the line length, but of course I can just put the parenthetical statement in-line. Done.